Versions-mac.plist.in

```diff
@@ -32,7 +32,10 @@
 	</dict>
 	<key>Features</key>
 	<array>
-		<string>feature-list</string>
+		<string>sgen-concurrent-gc</string>
+		<string>activation</string>
```

Comment:
What's `activation`? There hasn't been any activation-related changes in _ages_...

I think we should only add features that were introduced now (master/d15-1), since for older features you'll still need the version check (since those versions won't have Versions.plist).